src/components/movie-card.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { baseImageUrl } from "../services/instances";
+import { Box, Flex, Heading, Icon, Text } from "@chakra-ui/react";
+import { StarIcon } from "@chakra-ui/icons";
+
+export const MovieCard = (props) => {
+  console.log(props.image);
+  const bg = baseImageUrl + props.image;
+  return (
+    <>
+      <Box
+        color="white"
+        backgroundImage={`url(${bg})`}
+        bgPosition="top"
+        position="relative"
+        pt="160%"
```

Comment:
I'd rather we use fixed units, instead of percentage based ones. We can use the spacings chakra-ui provides for us
https://chakra-ui.com/docs/theming/theme#spacing

So if we say:
```js
<Box
  ...
  pt={12} // this would be equivalent to 3rem
/>
```

Reply:
No, I don't think so buddy